src/level.rs

```diff
@@ -0,0 +1,258 @@
+// Copyright 2014 The html5ever Project Developers. See the
```

Comment:
Sure. Included the second part in the cleanup PR: https://github.com/servo/unicode-bidi/pull/29